python/dataproc_templates/gcs/README.md

```diff
@@ -266,6 +266,81 @@ export JARS=<gcs-bucket-location-containing-jar-file>
     --gcs.jdbc.output.url="jdbc:mysql://12.345.678.9:3306/test?user=root&password=root"
 ```
 
+# GCS To MongoDB
+
+Template for reading files from Google Cloud Storage and writing them to a MongoDB Collection. It supports reading JSON, CSV, Parquet and Avro formats.
+
+## Arguments
+
+* `gcs.mongo.input.format`: Input file format (one of: avro,parquet,csv,json)
+* `gcs.mongo.input.location`: GCS location of the input files (format: `gs://BUCKET/...`)
+* `gcs.mongo.output.uri`: MongoDB Output URI for connection
+* `gcs.mongo.output.database`: MongoDB Output Database Name
+* `gcs.mongo.output.collection`: MongoDB Output Collection Name
+* `gcs.mongo.output.mode`: Output write mode (one of: append,overwrite,ignore,errorifexists)(Defaults to append)
+* `gcs.mongo.batch.size`: Output Batch Size (Defaults to 512)
+
+## Usage
+
+```
+$ python main.py --template GCSTOMONGO --help
+
+usage: main.py --template GCSTOMONGO [-h] \
+    --gcs.mongo.input.location GCS.MONGO.INPUT.LOCATION \
+    --gcs.mongo.input.format {avro,parquet,csv,json} \
+    --gcs.mongo.output.uri GCS.MONGO.OUTPUT.URI \
+    --gcs.mongo.output.database GCS.MONGO.OUTPUT.DATABASE \        
+    --gcs.mongo.output.collection GCS.MONGO.OUTPUT.COLLECTION \
+    --gcs.mongo.batch.size GCS.MONGO.BATCH.SIZE \
+    --gcs.mongo.output.mode {overwrite,append,ignore,errorifexists}
+
+
+optional arguments:
+  -h, --help            show this help message and exit
+  --gcs.mongo.input.location GCS.MONGO.INPUT.LOCATION
+                        GCS location of the input files
+  --gcs.mongo.input.format {avro,parquet,csv,json}
+                        Input file format (one of: avro,parquet,csv,json)                        
+  --gcs.mongo.output.uri GCS.MONGO.OUTPUT.URI
+                        Driver URI to connect with MongoDB Instance consisting of username and passwprd as well
+  --gcs.mongo.output.database GCS.MONGO.OUTPUT.DATABASE
+                        MongoDB Database Name
+  --gcs.mongo.output.collection GCS.MONGO.OUTPUT.COLLECTION
+                        MongoDB output Collection name
+  --gcs.mongo.batch.size GCS.MONGO.BATCH.SIZE
+                        MongoDB output Batch size
+  --gcs.mongo.output.mode {overwrite,append,ignore,errorifexists}
+                        Output write mode (one of: append,overwrite,ignore,errorifexists) (Defaults to append)                        
+```
+
+## Required JAR files
```

Comment:
If possible, please add wget commands to get these jars downloaded from maven repo.

Reply:
Added wget command to download the jar files.